docs/pages/setup/reference/config.mdx

```diff
@@ -35,8 +35,11 @@ teleport:
     # PID file for Teleport process
     #pid_file: /var/run/teleport.pid
 
-    # Invitation token used to join a cluster. it is not used on
-    # subsequent starts
+    # Invitation token or file path containing token used to join a cluster. it is not used on
+    # subsequent starts.  
+    # File path example:
```

Comment:
```suggestion
    #
    # File path example:
```
I liked how the original commit separated this from the main comment—it's a bit clearer this way.